shared/versions/stack/7.12.asciidoc

```diff
@@ -0,0 +1,50 @@
+:version:                7.12.0
+////
+bare_version never includes -alpha or -beta
+////
+:bare_version:           7.12.0
+:logstash_version:       7.12.0
+:elasticsearch_version:  7.12.0
+:kibana_version:         7.12.0
+:apm_server_version:     7.12.0
+:branch:                 7.x
```

Comment:
This should be 7.12 now, right?

Reply:
Yes. Fixed with 597151adcbde64bd7433a839653d940fe742b7f6